database/ci/Dockerfile

```diff
@@ -11,5 +11,5 @@ COPY ./liquibase/scripts/*.sh /docker-entrypoint-initdb.d/
 COPY ./liquibase/scripts/dbInit /docker-entrypoint-initdb.d/
 COPY ./liquibase/scripts/dbCi /docker-entrypoint-initdb.d/
 
-RUN curl "https://cida.usgs.gov/artifactory/nldi/datasets/nhdplus.yahara.pgdump.gz" -o $LIQUIBASE_HOME/nhdplus.yahara.pgdump.gz
-RUN curl "https://cida.usgs.gov/artifactory/nldi/datasets/characteristic_data.yahara.pgdump.gz" -o $LIQUIBASE_HOME/characteristic_data.yahara.pgdump.gz
+RUN curl "https://artifactory.wma.chs.usgs.gov/artifactory/wma-binaries/iow/nldi/nhdplus.yahara.pgdump.gz" -o $LIQUIBASE_HOME/nhdplus.yahara.pgdump.gz
```

Comment:
Need to use the public URL - https://artifactory.wma.usgs.gov

Reply:
Okay, done.  Note that travis failed once.  I then added a blank line to trigger another build, and travis passed.  It seems like the public artifactory is slower than the private one and can trigger random timeouts.